### PR TITLE
Feature/migrate footer to tailwind

### LINKED ISSUE #23: Tests are broken
Seems that some files are missing for the test script to work correctly:

```shell
[william@william-pc coding-coach]$ yarn test
yarn run v1.9.4
$ node scripts/custom-config test --env=jsdom
/home/william/Dev/Self/WNemencha/coding-coach/node_modules/proxyquire/lib/proxyquire.js:161
    throw err
    ^

Error: Cannot find module '../utils/createJestConfig' from '/home/william/Dev/Self/WNemencha/coding-coach/node_modules/react-scripts/scripts'
    at Function.module.exports [as sync] (/home/william/Dev/Self/WNemencha/coding-coach/node_modules/resolve/lib/sync.js:43:15)
    at Proxyquire._resolveModule (/home/william/Dev/Self/WNemencha/coding-coach/node_modules/proxyquire/lib/proxyquire.js:137:20)
    at Proxyquire.<anonymous> (/home/william/Dev/Self/WNemencha/coding-coach/node_modules/proxyquire/lib/proxyquire.js:205:35)
    at Array.reduce (<anonymous>)
    at Proxyquire._withoutCache (/home/william/Dev/Self/WNemencha/coding-coach/node_modules/proxyquire/lib/proxyquire.js:204:6)
    at Proxyquire.load (/home/william/Dev/Self/WNemencha/coding-coach/node_modules/proxyquire/lib/proxyquire.js:129:15)
    at Object.<anonymous> (/home/william/Dev/Self/WNemencha/coding-coach/scripts/custom-config.js:39:5)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

### LINKED ISSUE #34: Account Settings
**Acceptance Criteria:**

A user can update the following required fields:
- Name
- Email address
- Job title
- Languages the user speaks

A user can update the following optional fields:
- Social media accounts
- Company
- Skills to mentor in (depends on account type)
- Skills to be mentored in depends on account type)

Users will be able to set their account type to be mentor, mentee, or both.

A mentor can update the following fields:
- Skills they are able to mentor in
- Proficiency of skills they are able to mentor in

A mentee can update the following fields:
- Skills they are looking to improve

All users are able to set themselves as a mentor.

### LINKED ISSUE #42: Delete Account
**Acceptance Criteria:**
- Users can delete their accounts.
- This will remove all data from the database.
- All current mentorships will be notified.

### PR DESCRIPTION
**Pull request by sup-rith**
# Details

## Description

Eliminated all SCSS and added tailwindcss classes to footer.js

## Steps to QA

- [x] Step 1

## Issue

Closes #123

This closes #34, closes #23, and closes #42

### CODE REVIEW

**Comment by sup-rith** [comment]
This issue is ready to be reviewed @crysfel @emmawedekind , I'm not too sure on how to change the labels.

File: src/components/footer/Footer.js [hunk 1]
```diff
@@ -5,20 +5,20 @@ import Button from 'components/button/Button';
...
-  return (
-    <footer className={styles.footerMain}>
-      <div className={styles.footerInner}>
-        <div className={styles.column}>{/* <DonateButton /> */}</div>
+  const buttonClasses =
+    'inline-block p-1 text-xs secondary-lighter no-underline hover:secondary-dark';
 
+  return (
+    <footer className="container">
+      <div className="flex justify-end ml-auto mr-auto pl-8 pr-8 pt-4 pb-4 flex-col items-center border-t border-solid border-secondary-lightest md:flex-row md:ml-1.9375rem md:mr-1.9375rem">
```
[the remark this answers is not in the document]
Comment (trent-boyd) [reply src/components/footer/Footer.js]: Oh, here's a possible alternative. Backticks can be used to spread a string across as many lines as you like, and Prettier won't clobber your changes on save. Using code from `Hero.js` as an example:

``` js
  <h2
    className={`
    text-white text-sans text-2xl
    mb-4
    font-content font-hairline
    leading-normal
  `}
  >
```

[the remark this answers is not in the document]
Comment (crysfel) [reply src/components/footer/Footer.js]: I like backticks and grouping styles by category

[the remark this answers is not in the document]
Comment (tureey) [reply src/components/footer/Footer.js]: same

**Comment by crysfel** [comment]
Closing this due to inactivity

**Pull request closed without merging**